Mathlib/Data/Equiv/Functor.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2019 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Simon Hudon, Scott Morrison
+-/
+import Mathlib.Data.Equiv.Basic
+
+/-!
+
+# Functors can be applied to `equiv`s.
+
+```
+def functor.map_equiv (f : Type u → Type v) [functor f] [is_lawful_functor f] :
+  α ≃ β → f α ≃ f β
+```
+
+-/
+
+universes u v w
+
```

Comment:
```suggestion
```

Pre-declaring universes is mostly unnecessary these days.

Reply:
OK they're removed. It kind of feels counterintuitive to me to omit information.

Reply:
For me it is just meaningless boilerplate, even in Lean 3 it didn't make a semantic difference.  If you left it out, you got a compile error.  In Lean 4 it has even less effect.  The file compiles both with and without the line, and the end result is exactly the same.